src/test/external/ext_out_cluster_test.go

```diff
@@ -5,33 +5,42 @@
 package external
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
 	"io"
+	"mime/multipart"
 	"net/http"
+	"os"
+	"path/filepath"
 	"testing"
 
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
 	"github.com/defenseunicorns/zarf/src/pkg/utils/exec"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
+	"helm.sh/helm/v3/pkg/repo"
 )
 
 // Docker/k3d networking constants
 const (
-	network      = "k3d-k3s-external-test"
-	subnet       = "172.31.0.0/16"
-	gateway      = "172.31.0.1"
-	giteaIP      = "172.31.0.99"
-	giteaHost    = "gitea.localhost"
-	registryHost = "registry.localhost"
-	clusterName  = "zarf-external-test"
+	network       = "k3d-k3s-external-test"
+	subnet        = "172.31.0.0/16"
+	gateway       = "172.31.0.1"
+	giteaIP       = "172.31.0.99"
+	giteaHost     = "gitea.localhost"
+	registryHost  = "registry.localhost"
+	clusterName   = "zarf-external-test"
+	giteaUser     = "git-user"
+	giteaPassword = "superSecurePassword"
```

Comment:
```suggestion
	commonPassword = "superSecurePassword"
```
Technically this password is used between gitea and the registry